### ISSUE TITLE
Please stop ignoring issues

### ISSUE THREAD

**Issue by thany** [mapsme/omim#13327]
What will it take for the MAPS.ME team to start responding to issues, fixing them, and actually listening to their users?

Just take a look at the sheer number of issues. Every 3 months or so, when a new version is released, some BS feature is added that literaly *no one* asked for (if I'm wrong, please prove it) and yet pressing bugs and other shortcomings are mostly *completely* ignored.

Please respond.

**Comment by thany** [comment]
Closing this issue wil prove I'm right, btw. So doing that might not be a good idea. Because this too, happened in the past.

**Comment by FinixFighter** [comment]
Communication with users is essential and brings a lot of benefits. I agree with you. This app has so much potential!!

**Comment by jidanni** [comment]
Maybe they actually use a different issue tracker somewhere else?

**Comment by jidanni** [comment]
> Closing this issue wil prove I'm right, btw. So doing that might not be a good idea. Because this too, happened in the past.

Well at least they might not also lock it, like https://github.com/landley/toybox/issues/231

**Comment by thany** [comment]
> Maybe they actually use a different issue tracker somewhere else?

Yeah I think I remember them mentioning an internal roadmap, so that makes me believe there might be an internal issue tracker as well. It doesn't seem to contain much of any of the important bugs, or even the grossly overdue ones.

When a commercial company goes opensource, it can be an awesome thing. But as soon as they keep a hidden agenda, it becomes more an more viable to fork it and continue with a set of people that do care.

Or maybe I'm wrong and they work like maybe 1 hour a week on this project. That would explain things, too. Or something else is happenning that is completely understandable, if only they would be open about it.

**Comment by jidanni** [comment]
(Don't tell anybody, but it seems they indeed responded within a business day to my # 13533 and  # 13534 .)

**Comment by MagTun** [comment]
I agree, several time I have wanted to report a bug/do a feature request, just to find out that someone has already reported/asked this... but that was several years ago. And no update since.
